src/coprocessor/endpoint.rs

```diff
@@ -330,14 +332,17 @@ impl<E: Engine> Endpoint<E> {
         // When this function is being executed, it may be queued for a long time, so that
         // deadline may exceed.
         tracker.on_scheduled();
+        minitrace::new_span(TraceEvent::Scheduled).enter();
         tracker.req_ctx.deadline.check()?;
 
         // Safety: spawning this function using a `FuturePool` ensures that a TLS engine
         // exists.
+        let snapshot_span = minitrace::new_span(TraceEvent::Snapshot);
```

Comment:
need to `enter()`

Reply:
> need to `enter()`

`enter()` means corresponding span should be the parent span of next created one, here I don't want `snapshot_span` be the parent of any one else.